src/P/List.hs

```diff
@@ -45,3 +46,11 @@ lastMaybe = listToMaybe . reverse
 -- | count the number of elements satisfying a predicate in a list
 count :: (a -> Bool) -> [a] -> Int
 count predicate = length . filter predicate
+
+safeIndex :: [a] -> Int -> Maybe a
+safeIndex xs i
```

Comment:
fwiw it follows the same convention as `(!!)` http://hackage.haskell.org/package/base-4.9.0.0/docs/Prelude.html#v:-33--33-